README.md

```diff
@@ -1,63 +1,59 @@
+2. Top right click new Application, name it accordingly
   
 ![Discord Application Button](https://cdn.discordapp.com/attachments/631249406775132182/722901729573863424/eb9cd2edd04845b4a5a9581f7f897cc1.png)
 
-4. Now click on your new application you just created
+3. Now click on your new application you just created
 
-5. Select the Bot option on the left side of the screen
+4. Select the Bot option on the left side of the screen
 
 ![Discord Bot Option](https://cdn.discordapp.com/attachments/631249406775132182/722902298896105544/454736b9a5d5ce7e45dc2aedad8b8d34.png)
 
-6. Top right click add bot
+5. Top right click add bot
 
-7. Now you can see the token ID hidden, click copy and there you go you have your token ID
+6. Now you can see the token ID hidden, click copy and you have your token ID
 
 ![Token](https://cdn.discordapp.com/attachments/631249406775132182/722902431893028905/e6a8b8f3fae0e3f6650d7e5e6ac148b8.png)
 
 ## 2. How to set up the bot <h2>
-1. The first thing you want to fill out for it to work is config.json in the config Folder
+1. The first thing you want to fill out for it to work is `config.json` in the config Folder
   
 ![config file](https://cdn.discordapp.com/attachments/631249406775132182/722903468884623360/9374ffcdfbe7479e389cbbe079fb83d5.png)
 
-2. YOU MUST FILL OUT ALL ASPECTS FOR IT TO WORK
+2. **YOU MUST FILL OUT ALL ASPECTS FOR IT TO WORK**
 
 3. Starting with token, you want to put your bot token you just made into token area
 
-4. addRole: addRole is defaulted to false, if you put it to true make sure to put the role ID you want players who joins ur discord server to get.
+4. `addRole` : `addRole` is defaulted to `false`, if you put it to true make sure to put the role ID you want players who joins your discord server to get.
 
-5. serverName: just put ur server name I.E.(whateverMS, nottakenMS)
+5. `serverName` : Just put ur server name I.E.(whateverMS, nottakenMS)
 
-6. serverWebsite: insert your website URL.
+6. `serverWebsite` : Insert your website URL.
 
-7. serverMascot : URL of a picture, you kinda need to fill this one out otherwise you will get errors
+7. `serverMascot` : URL of a picture, you kinda need to fill this one out otherwise you will get errors
 
-7.5: serverColor : the hexcode to a color that matches your server artwork.
+7.5: `serverColor` : The hexcode to a color that matches your server artwork.
 
-8. databaseName : your database name.
+8. `databaseName` : Your database name.
 
-9. databasePass : Password of your database I defaulted it to nothing, but that is where you wanna put your SQL password.
+9. `databasePass` : Password of your database I defaulted it to nothing, but that is where you wanna put your SQL password.
 
-10. databaseUser: defaulted to root, change if otherwise.
+10. `databaseUser` : Defaulted to root, change if otherwise.
 
-11. databaseHost : again defaulted to localhost.
+11. `databaseHost` : Again defaulted to localhost.
```

Comment:
```suggestion
11. `databaseHost` : Defaults to `localhost`.
```

Reply:
See above note on `defaults` vs `defaulted`.
Would also suggest to drop the "again" for being redundant for a list form.